### ISSUE TITLE
Event and Task did not get WhatId and WhoId migrated

### ISSUE THREAD

**Issue by harshalsakpal** [forcedotcom/Data-Migration-Tool#9]
Hi,

I am trying to migrate Events and Tasks using the modified AccountwithAssetsAndOppties_hierarchical.json and following is my snippet. It did not migrate the Contact (WhoId) and Account (WhatId) in this case. Any idea why? In our org WhatId is always an Account and WhoId is always a Contact.

`
.... more objects before
{
 	"parentMappedField": "WhatId",
 	"childObject": "Event",
 	"ExternalIdField": "Account.External_Id__c",
 	"where": " Account.External_Id__c = 'MYEXTID-JIBBER-JABBER-SELF' AND IsChild = FALSE",
 	"unmappedFields": [
 		"AccountId"
 	],
 	"sequence": 2
 }, {
 	"parentMappedField": "WhatId",
 	"childObject": "Task",
 	"ExternalIdField": "Account.External_Id__c",
 	"where": " Account.External_Id__c = 'MYEXTID-JIBBER-JABBER-SELF'",
 	"unmappedFields": [
 		"AccountId"
 	],
 	"sequence": 3
 },
.... more objects after
`

**Comment by anoop-76** [comment]
@harshalsakpal : Have you already migrated accounts/contacts?

**Comment by harshalsakpal** [comment]
@anoop-76 Yes. Account, Opportunities, Leads migrated just fine. Event and Tasks too are getting migrated but without WhatId and WhoId association. I am not explicitly migrating Contacts because our managed package (Financial Services Cloud) triggers are creating Contact for every Account.
